.gitignore

```diff
@@ -8,6 +8,7 @@ _build.*
 *.os
 *.pyc
 *~
+*.swp
```

Comment:
I'm just curious why you found this necessary? What kind of files are these and what generates them?

Reply:
They are Vim “swap” files. See http://vimdoc.sourceforge.net/htmldoc/recover.html for example.